tests/Traits/JsonPathAccessorTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+
+namespace Softonic\GraphQL\Traits;
+
+use PHPUnit\Framework\TestCase;
+
+class TestObject {
```

Comment:
I think It would fit better the `TestClass` name

Reply:
Well it's because the class tested is ObjectAccessor, so to follow the same naming.